docs/samples/Microsoft.ML.Samples/Dynamic/KeyToValue_Term.cs

```diff
@@ -31,16 +31,16 @@ public static void KeyToValue_Term()
             // making use of default settings.
             string defaultColumnName = "DefaultKeys";
             // REVIEW create through the catalog extension
-            var default_pipeline = new WordTokenizingEstimator(ml, "Review")
-                .Append(new ValueToKeyMappingEstimator(ml, "Review", defaultColumnName));
+            var default_pipeline = mlContext.Transforms.Text.TokenizeWords("Review")
```

Comment:
>default_pipeline [](start = 16, length = 16)

Ugh. Our samples have been written with Python variable naming conventions, not C# ones? I'll have to file an issue about this... #2155.